onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -111,55 +113,24 @@ def create_backend_test(testname=None):
     else:
         # Tests that are failing temporarily and should be fixed
         current_failing_tests = [#'^test_cast_STRING_to_FLOAT_cpu',  # old test data that is bad on Linux CI builds
-                                 '^test_qlinearconv_cpu',
-                                 '^test_gru_seq_length_cpu',
-                                 '^test_dynamicquantizelinear_expanded.*',
-                                 '^test_dynamicquantizelinear_max_adjusted_expanded.*',
-                                 '^test_dynamicquantizelinear_min_adjusted_expanded.*',
-                                 '^test_top_k.*',
                                  '^test_unique_not_sorted_without_axis_cpu', # bad expected data. enable after https://github.com/onnx/onnx/pull/2381 is picked up
                                  '^test_mod_float_mixed_sign_example_cpu', #onnxruntime::Mod::Compute fmod_ was false. fmod attribute must be true for float, float16 and double types
-                                 '^test_shrink_cpu', #Invalid rank for input: x Got: 1 Expected: 2 Please fix either the inputs or the model.
```

Comment:
invalid input shape/rank now result in warnings only (don't error out). So this test can be re-enabled.